sentry_streams/src/filter_step.rs

```diff
@@ -0,0 +1,70 @@
+use crate::routes::{Route, RoutedValue};
+use pyo3::{Py, PyAny, Python};
+use sentry_arroyo::processing::strategies::{
+    CommitRequest, ProcessingStrategy, StrategyError, SubmitError,
+};
+use sentry_arroyo::types::Message;
+use std::time::Duration;
+
+pub struct Filter {
+    pub callable: Py<PyAny>,
+    pub next_step: Box<dyn ProcessingStrategy<RoutedValue>>,
+    pub route: Route,
+}
+
+impl Filter {
+    /// A strategy that takes a callable, and applies it to messages
+    /// to either filter the message out or submit it to the next step.
+    /// The callable is expected to take a Message<Py<PyAny>>
```

Comment:
It takes `Message<RoutedValue>` not `Message<Py<PyAny>>`

Reply:
Right, I wasn't sure if we were referring to the user-defined function (which is currently in Python) or the `callable` which only assumes a single argument of `Py<PyAny>`